app/src/main/java/org/mozilla/focus/web/GeckoWebViewProvider.java

```diff
@@ -43,26 +48,27 @@
 /**
  * WebViewProvider implementation for creating a Gecko based implementation of IWebView.
  */
-public class WebViewProvider {
+public class GeckoWebViewProvider implements IWebViewProvider {
```

Comment:
\[PMD\] INFO: The class 'GeckoWebViewProvider' has a NCSS line count of 290 \(Highest = 6\)\.